buildlib/push-release-images.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -eE
```

Comment:
where/how is this script used?

Reply:
Please see https://github.com/openucx/ucx/pull/5184/files#diff-5725f980a18c27ae475c32ff3a5d07fb.

Reply:
this is doc file, 
who should run this script? is it when adding new matrix dimension?

Reply:
This script is for docker images maintainer, anyone with write permissions to Harbor registry can run it.